services/graph/pkg/service/v0/links.go

```diff
@@ -202,7 +202,7 @@ func (g Graph) libreGraphPermissionFromCS3PublicShare(createdLink *link.PublicSh
 }
 
 func parseAndFillUpTime(t *time.Time) *types.Timestamp {
```

Comment:
@rhafer @fschade This should also apply to the normal sharing link expiration.

Reply:
@micbar So we want to limit the granularity of share expiration to a day? Didn't know that. Shouldn't that be done in the CS3 shareprovider then?

Reply:
@micbar needs discussion, the isZero check is valid and needed... but the rounding seems to be something we should talk about, the api allows setting not just the day but instead also allows to define the exact time ... so no rounding is needed, but we could be wrong, you have some minutes to talk about it?